src/table/toolbar/table-toolbar.component.ts

```diff
@@ -92,6 +101,7 @@ export class TableToolbar {
 	}
 
 	onCancel() {
+		this.onCancelSelected.emit();
 		for (let i = 0; i < this.model.rowsSelected.length; i++) {
 			this.model.selectRow(i, false);
```

Comment:
@Cal Do you think it would break anything if the whole loop is changed into `this.model.selectAll(false)`?

Reply:
Nope, that's definitely how we should be handling it now